<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Send_Test">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>278</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Test</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_Final">
    <property name="geometry">
     <rect>
      <x>593</x>
      <y>278</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_path_file">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_path">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="image_path">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add_path_image">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_path_csv">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="csv_path">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Attachment Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add your message here. Use {} where you would normally add the name

</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_num">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Ph. Number:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_file">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_image">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="message">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>321</width>
      <height>70</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add_path_selenium">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Chromedriver:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selenium_path">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
